<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Discord Custom Status</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>294</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="centralLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>ClientID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="clientIDLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Details</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="detailsLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>State</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="stateLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Start Timestamp</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="startTimestampLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>End Timestamp</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="endTimestampLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Large Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="largeImageLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>Large Tooltip</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="largeTooltipLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Small Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="smallImageLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Small Tooltip</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="smallTooltipLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="changePushButton">
         <property name="text">
          <string>Change</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
